corpora/csip/metadata/mets-root/CSIP5/testCase.xml

```diff
@@ -1,23 +1,88 @@
-<!-- Root element for an individual test case, allows these to be wrapped into
-XML lists -->
-<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
-  testable="UNKNOWN">
-  <!-- Unique ID for the test case -->
-  <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP5"/>
-  <!-- URL references to requirements for convenient lookup -->
+<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd" testable="TRUE">
+  <id specification="CSIP" version="2.0.3" requirementId="CSIP5"/>
```

Comment:
2.0.4 is the latest

Reply:
Fixed.

Reply:
It says 2.0.3 and not 2.0.4?

Reply:
I think I get the issue now: GitHub keeps linking to the version that was valid at the moment of initial review, so the updates made by the later commit 2875094fc9c609c7eed60194ee9f715eec595635 are not there. It seems the "Files changed" tab of the PR always displays the latest state.